docs/tutorials/getting-started.md

```diff
@@ -1,5 +1,3 @@
-# Getting Started: Deploy MicroCeph via Charm
-
 In this guide we will deploy a 3-node charm-microceph cluster in an LXD environment using Juju. This guide assumes setting up an LXD environment on a single host machine with simulated storage. This works great for learning and testing purposes; in a production environment you would typically utilize Juju to deploy to several physical machines and storage media. 
```

Comment:
This is under tutorial, from here onwards in other sections, we should refer it as a tutorial rather than a guide.